client/src/components/relatedItems/relatedCarousel/relatedCard/cardComponents/countStarRatings.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import _ from 'underscore';
+import StarRating from '../../../../ratingsReviews/StarRating.jsx';
+
+function CountRatings({ ratings }) {
+  const ratingCount = _.reduce(
```

Comment:
Love your use of reduce. I will look to implement the same in my star rating component.

Reply:
got it from Kate haha. shoutout to Kate!